src/httpclient.c

```diff
@@ -452,7 +452,8 @@ static void usage(const char *progname)
             "Usage: %s [options] <url>\n"
             "Options:\n"
             "  -2 <ratio>   HTTP/2 ratio (between 0 and 100)\n"
-            "  -3           HTTP/3-only mode\n"
+            "  -3 <ratio>   HTTP/3 ratio (between 0 and 100)\n"
+            "               `ratio` can be omitted, implies `-3 100`\n"
```

Comment:
```suggestion
```
How about removing this line? It is specific to glibc, and I do not think we want to endorse use of this feature. The only reason we allow ratio to be omitted is to support existing users using linux.